nucleus/dataset.py

```diff
@@ -211,6 +214,11 @@ def append(
         """
         check_for_duplicate_reference_ids(dataset_items)
 
+        if len(dataset_items) > WARN_FOR_LARGE_UPLOAD and not asynchronous:
+            print(
+                "WARNING: for large uploads we recommend uploading your images remotely to a cloud storage provider and then using asynchronous=True."
```

Comment:
I would rewrite this as:

"Tip: for large uploads, get faster performance by importing your images into Nucleus directly from a cloud storage provider. See https://dashboard.scale.com/nucleus/docs/api?language=python#guide-for-large-ingestions for details."

Reply:
Thanks!